views/posts/index.hbs

```diff
@@ -2,21 +2,45 @@
 <h5 class="text-center"><a class="new-post-link" href="/posts/new">New post</a></h5>
 <div class="container" id="posts">
   {{#each posts}}
-  <div class="row no-gutters border rounded overflow-hidden flex-md-row mb-4 shadow-sm h-md-250 position-relative post" id="{{@index}}">
+  <div class="post-container mb-4">
```

Comment:
This is the reason why the integration test for posts is failing. The test expects a div with a class containing 'post' directrly unser the container with the id 'posts'.
But now there is an additional div with the class 'post-container mb-4' between the two.